alias_test.go

```diff
@@ -0,0 +1,55 @@
+package main_test
+
+import (
+	"context"
+	"testing"
+	"strings"
+
+	"./db"
```

Comment:
Since swapping to Go modules, we are using the `github.com/uclaacm/teach-la-go-backend/db` style of imports.

Reply:
Updated to import from GitHub